event-bus/shared/src/main/java/com/github/dynamicextensionsalfresco/event/impl/Activator.java

```diff
@@ -0,0 +1,22 @@
+package com.github.dynamicextensionsalfresco.event.impl;
+
+import com.github.dynamicextensionsalfresco.event.EventBus;
+import kotlin.jvm.internal.Intrinsics;
+import org.jetbrains.annotations.NotNull;
+import org.osgi.framework.BundleActivator;
+import org.osgi.framework.BundleContext;
+
+/**
+ * @author Laurent Van der Linden
+ */
+public final class Activator implements BundleActivator {
+
+    public void start(@NotNull BundleContext context) {
```

Comment:
The Kotlin implementation is using the `@Throws(Exception::class)` annotation to signal a checked exception can be thrown

Reply:
Which will never be thrown in this implementation? The interface (`BundleActivator`) still has it.

I agree that this is not a literal conversion. If you want me to change it anyway, I will.

Reply:
Fair point :-)